manifests/crane/params.pp

```diff
@@ -6,6 +6,6 @@
   $debug = false
   $port = 5000
 
-  $data_dir = '/var/lib/crane/metadata'
+  $data_dir = '/var/lib/pulp/published/docker/v2/app'
```

Comment:
so i think this was set that way because that is the default in crane: https://github.com/pulp/crane/blob/master/crane/data/default_config.conf

we likely need to override this parameter.